haskala/modules/custom/haskala_book/plugins/content_types/books_list/books_list.inc

```diff
@@ -72,8 +75,19 @@ function haskala_book_books_list_content_type_render($subtype, $conf, $args, $co
     $block_vars['class'] = in_array($letter, $letters_he) ? 'he' : 'en';
     $block_vars['books_list'] = '';
     foreach ($block as $node) {
-       $view = node_view($node, 'teaser');
-       $block_vars['books_list'] .= drupal_render($view);
+      if ($node_type == 'person') {
+        if (in_array($letter, $letters_en)) {
+          $view = node_view($node, 'teaser_en');
+        }
+        else if (in_array($letter, $letters_he)) {
+          $view = node_view($node, 'teaser_he');
+        }
+      }
+      elseif ($node_type == 'book') {
+        $view = node_view($node, 'teaser');
```

Comment:
Maybe would be neater to call node_view() once, and have the second parameter as a variable